cmd/bashbrew/git.go

```diff
@@ -73,6 +76,9 @@ func ensureGitInit() error {
 	config.Raw = config.Raw.SetOption("gc", "", "auto", "0")
 	gitRepo.Storer.SetConfig(config)
 
+	netrcClient := &http.Client{Transport: &netrcTransport{}}
+	goGitClient.InstallProtocol("https", goGitHttp.NewClient(netrcClient))
```

Comment:
Based on the custom_http example: https://github.com/go-git/go-git/blob/838eb2979f6085fc0e5bc8856e7f1944a455a970/_examples/custom_http/main.go#L39